client/components/jetpack/scan-threats-new/index.tsx

```diff
@@ -0,0 +1,347 @@
+		headerSummary = prefix + ' ' + headerSummary;
+	}
+	if ( suffix ) {
+		headerSummary += ' ' + suffix;
+	}
+
+	return headerSummary;
+};
+
+const ScanThreats = ( { error, site, threats }: Props ) => {
+	const {
+		updatingThreats,
+		selectedThreat,
+		setSelectedThreat,
+		fixThreats,
+		updateThreat,
+	} = useThreats( site.ID );
+	const [ showThreatDialog, setShowThreatDialog ] = React.useState( false );
+	const [ showFixAllThreatsDialog, setShowFixAllThreatsDialog ] = React.useState( false );
+	const [ actionToPerform, setActionToPerform ] = React.useState< ThreatAction >( 'fix' );
+	const dispatch = useDispatch();
+	const dispatchScanRun = React.useCallback( () => {
+		triggerScanRun( site.ID )( dispatch );
+	}, [ dispatch, site ] );
+
+	const allFixableThreats = threats.filter(
+		( threat ): threat is FixableThreat =>
+			threat.fixable !== false && threat.fixerStatus !== 'in_progress'
+	);
+	const hasFixableThreats = !! allFixableThreats.length;
+
+	const openFixAllThreatsDialog = React.useCallback( () => {
+		dispatch(
+			recordTracksEvent( `calypso_jetpack_scan_allthreats_open`, {
+				site_id: site.ID,
+			} )
+		);
+		setShowFixAllThreatsDialog( true );
+	}, [ dispatch, site ] );
+
+	const openDialog = React.useCallback(
+		( action: ThreatAction, threat: Threat ) => {
+			const eventName =
+				action === 'fix'
+					? 'calypso_jetpack_scan_fixthreat_dialogopen'
+					: 'calypso_jetpack_scan_ignorethreat_dialogopen';
+			dispatch(
+				recordTracksEvent( eventName, {
+					site_id: site.ID,
+					threat_signature: threat.signature,
+				} )
+			);
+			setSelectedThreat( threat );
+			setActionToPerform( action );
+			setShowThreatDialog( true );
+		},
+		[ dispatch, setSelectedThreat, site ]
+	);
+
+	const closeDialog = React.useCallback( () => {
+		setShowThreatDialog( false );
+	}, [] );
+
+	const confirmAction = React.useCallback( () => {
+		closeDialog();
+		updateThreat( actionToPerform );
+	}, [ actionToPerform, closeDialog, updateThreat ] );
+
+	const confirmFixAllThreats = React.useCallback(
+		( fixableThreats ) => {
+			setShowFixAllThreatsDialog( false );
+			fixThreats( fixableThreats );
+		},
+		[ fixThreats ]
+	);
+
+	const isFixing = React.useCallback(
+		( threat: Threat ) => {
+			return (
+				!! updatingThreats.find( ( threatId ) => threatId === threat.id ) ||
+				threat.fixerStatus === 'in_progress'
+			);
+		},
+		[ updatingThreats ]
+	);
+
+	const maxSeverity = threats.reduce( ( max, threat ) => Math.max( max, threat.severity ), 0 );
+	const countMap = {
+		low: threats.filter( ( threat ) => threat.severity < 3 ).length,
+		high: threats.filter( ( threat ) => threat.severity >= 3 ).length,
+	};
+	const countMapFixable = {
+		low: allFixableThreats.filter( ( threat ) => threat.severity < 3 ).length,
+		high: allFixableThreats.filter( ( threat ) => threat.severity >= 3 ).length,
+	};
+
+	const headerSummary = getThreatCountMessage(
+		countMap.high,
+		countMap.low,
+		null,
+		translate( 'found' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **39** times:
`translate( 'Found' )` *ES Score: 7*